zynqmp/optee-client/optee-client.bb

```diff
@@ -0,0 +1,57 @@
+OPTEE_VERSION ??= "latest"
+SRCREV ??= "${AUTOREV}"
+BRANCH ??= "master"
+
+DESCRIPTION = "OP-TEE Client"
+
+#defined as closed license to prevent MD5 checksum verification
+LICENSE = "CLOSED"
+
+PROVIDES = "virtual/optee-client"
+
+inherit deploy
+
+DEPENDS += "virtual/optee-os"
+
+S = "${WORKDIR}/git"
+B = "${WORKDIR}/build"
+
+PV = "${OPTEE_VERSION}+git${SRCPV}"
+
+REPO ??= "git://github.com/OP-TEE/optee_client.git;protocol=https"
+BRANCHARG = "${@['nobranch=1', 'branch=${BRANCH}'][d.getVar('BRANCH', True) != '']}"
+SRC_URI = "${REPO};${BRANCHARG} \
+	"
+
+OPTEE_BASE_NAME ?= "${PN}-${PKGE}-${PKGV}-${PKGR}-${DATETIME}"
+OPTEE_BASE_NAME[vardepsexclude] = "DATETIME"
+
+COMPATIBLE_MACHINE = "zynqmp"
+PLATFORM_zynqmp = "zynqmp"
+
+# requires CROSS_COMPILE set by hand as there is no configure script
+export CROSS_COMPILE="${TARGET_PREFIX}"
+
+EXPORT_DIR = "${TMPDIR}/deploy/images/${MACHINE}/optee/export_client"
+
+EXTRA_OEMAKE_append = " CROSS_COMPILE=${CROSS_COMPILE}"
+EXTRA_OEMAKE_append = " DESTDIR=${EXPORT_DIR}"
+EXTRA_OEMAKE_append = " SBINDIR=/sbin"
+EXTRA_OEMAKE_append = " LIBDIR=/lib"
+EXTRA_OEMAKE_append = " INCLUDEDIR=/include"
+
+do_configure() {
+	oe_runmake clean -C ${S} clean
```

Comment:
minor: looks a strange configure way.

Reply:
What do you mean?

Reply:
configure is for configuring. cleaning for cleaning.

Reply:
this may be nitpicking. feel free to not address my comment.

Reply:
My bitbake recipes are based on TF-A recipe from xilinx-meta layer. Don't know why they did that, maybe its due to Petalinux. As i don't know what might happen if i change this, i prefer not to modify it.

Reply:
@mimok https://en.wikipedia.org/wiki/Cargo_cult_programming ;-)

Reply:
Sure I'm not a BB guru. However I find out that the default behaviour of `do_configure` is to perform a `make clean`. I'm going to check if I can remove that as this seems to be the yocto default behaviour.